services/diversity/src/routes/richness.js

```diff
@@ -153,5 +153,69 @@ module.exports = (errorHandler, Richness) => {
       });
   }));
 
+  /**
+   * @apiGroup s_richness
+   * @api {get} /richness/number-species/layer NOSLayer
+   * @apiName NOSLayer
+   * @apiVersion 1.0.0
+   * @apiDescription
+   * Layer of a specific group for richness - number of species in a given area. Parameter group
+   * may be selected from: total, endemic, invasive and threatened.
+   *
+   * @apiParam (Query params) {String} areaType area type
+   * @apiParam (Query params) {String|Number} areaId area id
+   * @apiParam (Query params) {String} group to select the proper layer
```

Comment:
You should document the options here as you did in the the other endpoint

Reply:
Done!